src/components/exchange/ExchangeDetailsRow.tsx

```diff
@@ -105,7 +94,7 @@ export default function ExchangeDetailsRow({
   ]);
 
   return (
-    <Container {...props}>
+    <Box alignItems="center" height="64px" justifyContent="center" width="full">
```

Comment:
this one was 60 before 
![image](https://user-images.githubusercontent.com/12115171/186046933-990e3247-6235-4a17-9671-d97adf9ee613.png)

Reply:
Im using 64 as its the closest number in our design system spacing scale, should I hard code it to 60px?

Reply:
i've been doing 

```
height={{ custom: 60 }
```

just to maintain the height